schemas/IlpHeader.json

```diff
@@ -6,11 +6,11 @@
   "properties": {
     "ledger": {
       "description": "The ledger where the transfer will take place",
-      "$ref": "Iri.json"
+      "type": "string"
     },
     "account": {
       "description": "Account holding the funds",
-      "$ref": "Iri.json"
+      "type": "string"
```

Comment:
I think we should have an `Address` schema that gives the type as well as a regex